hipify-clang/src/ArgParse.h

```diff
@@ -33,6 +33,8 @@ extern cl::opt<std::string> OutputFilename;
 extern cl::opt<std::string> OutputDir;
 extern cl::opt<std::string> TemporaryDir;
 extern cl::opt<std::string> CudaPath;
+extern cl::list<std::string> I;
```

Comment:
I suggest to rename these two variables to something like IncludeDirs and MacroNames since they are global and one-letter names can intersect with local variables

Reply:
Well, the idea was to mimic to clang's options in order not to set options separator '--' for specifying those options ('-I' and '-D') for clang. Moreover, those options become clang's options. As for local variables, they will differ anyway.

Reply:
Ok, I got what you meant. Let's they be IncludeDirs and MacroNames.